README.ko.md

```diff
@@ -0,0 +1,89 @@
+# caver-js-examples
+**caver-js-examples** 프로젝트는 SDK를 사용하는 분들이 좀 더 쉽고 빠르게 SDK를 사용하실 수 있게끔 도움을 드리기 위한 프로젝트입니다.
+다양한 상황에 맞는 예제코드를 직접 확인하시고 실행해보실 수 있습니다.
+
+## 1. 선행되어야 하는 작업
+### 1.1 실습을 위해 필요한 소프트웨어 설치
+> 본 프로젝트를 구성할 때 사용한 소프트웨어들을 기술합니다. 원활한 실습 진행을 위해 아래 기술한 소프트웨어들은 실습 전에 설치 해주시기 바랍니다.
+* [Node.js](https://nodejs.org/en/download/) (아래 버전 중 하나를 권장드립니다.)
+    * lts/erbium ([12.21.0](https://nodejs.org/dist/latest-v12.x/))
+    * lts/fermium ([14.16.0](https://nodejs.org/dist/latest-v14.x/))
+* [npm](https://www.npmjs.com/get-npm)
+* [gcc-c++](https://gcc.gnu.org/)
```

Comment:
```suggestion
본 프로젝트를 구성할 때 필요한 소프트웨어들을 기술합니다. 원활한 실습 진행을 위해 서는 [문서](https://ko.docs.klaytn.com/bapp/sdk/caver-js/getting-started#prerequisites)에 기술되어 있는 소프트웨어들은 실습 전에 설치해야 합니다.
```
If not essential, how about using docs link to reduce duplicate contents for easier maintenance?

Reply:
@jimni1222 
Good idea :)